### ISSUE TITLE
Warning: taxomony id doesn't exists for NC_001224.1!

### ISSUE THREAD

**Issue by novitch** [infphilo/centrifuge#52]
Hi, I ran into an issue when I tried to add Saccharomyces cerevisiae genomes and others into my centrifuge database. I have a warning of 55 542 sequences ID.
I created my own file "seqid2taxid.map" and I can grep the sequencesID on it, for instance in vim the sequence id NC_001224.1 is associated to tax id 559292 (tab separated and nothing other characters in the line ie: NC_001224.1^I559292$

Then I looked in names.dmp and node.dmp files and 559292 is associated to 
_grep -w "^559292" taxonomy/names.dmp_ 

559292  |       Saccharomyces cerevisiae S288c  |               |       scientific name |

And on nodes.dmp:

_grep -w "^559292" taxonomy/nodes.dmp_
559292  |       4932    |       no rank |               |       4       |       1       |       1       |       1       |       3       |       1       |       1       |       0       |               |

Could you explained if I am doing something wrong that I can correct because I really want to count S. cerevisae in my data and I can't associate taxonomy to their sequences.

Thanks,

Alban

**Comment by novitch** [comment]
I did other test and when I tried to build a database with only S.cerevisae and my .map file I obtained the same warnings while if I grep in my .map file the seqID of S.cerevisae into a new file and rerun the process with this new file as my .map, i have no warnings.

My .map file has 2 972 918 references because I would like to use scaffold genomes. Is it too much to make the centrifuge-build working?
